### ISSUE TITLE
History of polyarteritis nodosa

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10057]
**Preferred term label:**
History of polyarteritis nodosa

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cholecystocutaneous fistula

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
a PMH of specific diseases is outside the scope of HPO and would be encoded using phenopackets and Mondo terms.
